zum-back-end/api/index.js

```diff
@@ -0,0 +1,157 @@
+const fs = require("fs");
+const crypto = require("crypto");
+const puppeteer = require("puppeteer");
+const express = require("express");
+const router = express.Router();
+
+/* 랭킹 데이터 */
+router.get("/best", (req, res, next) => {
+  try {
+    let data = fs.readFileSync("cache/json/best.json");
+    data = JSON.parse(data);
```

Comment:
```suggestion
    const data = JSON.parse(fs.readFileSync("cache/json/best.json"));
```
이렇게 묶어버려도 괜찮을 것 같습니다.